src/app/public/src/fixture-types/toast/toast-fixture.spec.ts

```diff
@@ -0,0 +1,137 @@
+//#region imports
+
+import {
+  Component
+} from '@angular/core';
+
+import {
+  NoopAnimationsModule
+} from '@angular/platform-browser/animations';
+
+import {
+  TestBed
+} from '@angular/core/testing';
+
+import {
+  SkyToastModule,
+  SkyToastType
+} from '@skyux/toast';
+
+import {
+  SkyTestComponentSelector
+} from '../../component-selector';
+
+//#endregion imports
+
+//#region Test component
+@Component({
+  selector: 'toast-test',
+  template: `
+<sky-toast
+  [toastType]="toastTypeValue"
+  (closed)="onClosed()"
+  data-sky-id="test-toast"
+>
+  This is a sample toast.
+</sky-toast>
+  `
+})
+class TestComponent {
+  public set toastType(value: string) {
+    switch (value) {
+      case 'danger':
+        this.toastTypeValue = SkyToastType.Danger;
+        break;
+      case 'info':
+        this.toastTypeValue = SkyToastType.Info;
+        break;
+      case 'success':
+        this.toastTypeValue = SkyToastType.Success;
+        break;
+      case 'warning':
+        this.toastTypeValue = SkyToastType.Warning;
+        break;
+      default:
+        this.toastTypeValue = undefined;
+        break;
+    }
+  }
+
+  public toastTypeValue: SkyToastType;
+
+  public onClosed(): void { }
+}
+//#endregion Test component
+
+describe('Toast fixture', () => {
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        TestComponent
+      ],
+      imports: [
+        SkyToastModule,
+        NoopAnimationsModule
+      ]
+    });
+  });
+
+  it('should expose the expected properties', () => {
+    const fixture = TestBed.createComponent(
+      TestComponent
+    );
+
+    fixture.detectChanges();
+
+    const toast = SkyTestComponentSelector.selectToast(
+      fixture,
+      'test-toast'
+    );
+
+    expect(toast.text).toBe('This is a sample toast.');
+
+    const validToastTypes = [
+      'info',
+      'success',
+      'warning',
+      'danger'
+    ];
+
+    for (const validToastType of validToastTypes) {
+      fixture.componentInstance.toastType = validToastType;
+
+      fixture.detectChanges();
+
+      expect(toast.toastType).toBe(validToastType);
+    }
+
+    fixture.componentInstance.toastType = 'invalid';
+
+    fixture.detectChanges();
+
+    expect(toast.toastType).toBe('info');
```

Comment:
does setting it to undefined default to info?

Reply:
It does, because of the toast component's internal implementation. https://github.com/blackbaud/skyux-toast/blob/master/src/app/public/modules/toast/toast.component.ts#L41